Testing/CASLTests/Devices/Exceptions/AudioDeviceManagerNotInitializedExceptionTests.cs

```diff
@@ -21,31 +22,37 @@ public void Ctor_WhenInvokedWithNoParam_CorrectlySetsMessage()
         var exception = new AudioDeviceManagerNotInitializedException();
 
         // Assert
-        Assert.Equal("The audio device manager has not been initialized.", exception.Message);
+        var expectedMessage = "The audio device manager has not been initialized.";
+        exception.Message.Should().Be(expectedMessage);
     }
 
     [Fact]
     public void Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMesage()
```

Comment:
Whoops!!  I just caught a spelling mistake!!

Could you possibly fix this real quick?

```suggestion
    public void Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMessage()
```

I actually did a search in the entire code base and found multiple instances of this spelling mistake.  I had a hunch that it was copied from exception tests to other exception tests and I was correct.  I found 6 other instances of this unit test name with the same spelling mistake.

If you just do a search for `Ctor_WhenInvokedWithSingleMessageParam_CorrectlySetsMesage`, you will find them all.

Reply:
Fixed the spelling mistake on all occurrences!